icu4c/source/common/common.vcxproj

```diff
@@ -8,7 +8,7 @@
     <UseOfMfc>false</UseOfMfc>
```

Comment:
Since this is supposed to be building a static library, shouldn't the `ConfigurationType` be changed from `DynamicLibrary` to `StaticLibrary` ?
(For this vcxproj file, and for the other ICU library projects. Ex: i18n, io )